### PR TITLE
allow to be case insensitive inside zip

### PR DESCRIPTION
**Pull request by mbrocchieri**

### CODE REVIEW

File: fastexcel-reader/src/main/java/org/dhatim/fastexcel/reader/OPCPackage.java [hunk 1]
```diff
@@ -162,7 +163,18 @@ private InputStream getEntryContent(String name) throws IOException {
         }
         ZipArchiveEntry entry = zip.getEntry(name);
         if (entry == null) {
-            return null;
+            // to be case insensitive
+            Enumeration<ZipArchiveEntry> entries = zip.getEntries();
+            while (entries.hasMoreElements()) {
+                ZipArchiveEntry e = entries.nextElement();
+                if (e.getName().equalsIgnoreCase(name)) {
+                    entry = e;
```
Comment (aousadou) [inline fastexcel-reader/src/main/java/org/dhatim/fastexcel/reader/OPCPackage.java:171]: I suggest to return zip.getInputStream(e) here and remove condition after loop and directly return null if we are out of this loop without returning

**Review by aousadou — CHANGES_REQUESTED** [review]

File: fastexcel-reader/src/main/java/org/dhatim/fastexcel/reader/OPCPackage.java (hunk 1 above)
Comment (mbrocchieri) [reply fastexcel-reader/src/main/java/org/dhatim/fastexcel/reader/OPCPackage.java]: fixed

**Review by aousadou — APPROVED** [review]

File: fastexcel-reader/src/test/java/org/dhatim/fastexcel/reader/SimpleReaderTest.java [hunk 2]
```diff
@@ -134,4 +134,21 @@ public void testDefaultWorkbookPath2() throws IOException {
             }
         }
     }
+
+    @Test
+    public void testCaseInsensitiveInFileNames() throws IOException {
+        try (InputStream is = Resources.open("/xlsx/caseInsensitive.xlsx");
```
Comment (ochedru) [inline fastexcel-reader/src/test/java/org/dhatim/fastexcel/reader/SimpleReaderTest.java:140]: Is this file missing from the PR?

File: fastexcel-reader/src/test/java/org/dhatim/fastexcel/reader/SimpleReaderTest.java [hunk 3]
```diff
@@ -134,4 +134,21 @@ public void testDefaultWorkbookPath2() throws IOException {
...
+                assertTrue(it.hasNext());
+                Iterator<Cell> cellIt = it.next().iterator();
+                assertTrue(cellIt.hasNext());
+                Cell cell = cellIt.next();
+                assertEquals(CellType.STRING, cell.getType());
+                assertEquals("A", cell.getValue());
+            }
+        }
+    }
 }
```
Comment (ochedru) [inline fastexcel-reader/src/test/java/org/dhatim/fastexcel/reader/SimpleReaderTest.java:154]: I cannot see anything case-related in this test. Did I miss something?

Comment (mbrocchieri) [reply fastexcel-reader/src/test/java/org/dhatim/fastexcel/reader/SimpleReaderTest.java:154]: To understand what need to be case insentive for some xlsx files, please look at inside xlsx file
```
mathieu@localhost:~$ unzip caseInsensitive.xlsx
Archive:  caseInsensitive.xlsx
  inflating: [Content_Types].xml     
   creating: docProps/
  inflating: docProps/core.xml       
  inflating: docProps/app.xml        
   creating: _rels/
  inflating: _rels/.rels             
   creating: xl/
  inflating: xl/workbook.xml         
  inflating: xl/styles.xml           
  inflating: xl/sharedStrings.xml    
   creating: xl/_rels/
  inflating: xl/_rels/workbook.xml.rels  
   creating: xl/worksheets/
  inflating: xl/worksheets/sheet1.xml  

mathieu@localhost:~$ cat [Content_Types].xml
<?xml version="1.0" encoding="UTF-8"?>
<Types xmlns="http://schemas.openxmlformats.org/package/2006/content-types"><Override PartName="/_rels/.rels" ContentType="application/vnd.openxmlformats-package.relationships+xml"/><Override PartName="/docProps/app.xml" ContentType="application/vnd.openxmlformats-officedocument.extended-properties+xml"/><Override PartName="/docProps/core.xml" ContentType="application/vnd.openxmlformats-package.core-properties+xml"/><Override PartName="/xl/_rels/workbook.xml.rels" ContentType="application/vnd.openxmlformats-package.relationships+xml"/><Override PartName="/xl/sharedstrings.xml" ContentType="application/vnd.openxmlformats-officedocument.spreadsheetml.sharedStrings+xml"/><Override PartName="/xl/worksheets/sheet1.xml" ContentType="application/vnd.openxmlformats-officedocument.spreadsheetml.worksheet+xml"/><Override PartName="/xl/styles.xml" ContentType="application/vnd.openxmlformats-officedocument.spreadsheetml.styles+xml"/><Override PartName="/xl/workbook.xml" ContentType="application/vnd.openxmlformats-officedocument.spreadsheetml.sheet.main+xml"/>
</Types>
```

xl/sharedStrings.xml  file got S, but in [Content_Types].xml we got 
`<Override PartName="/xl/sharedstrings.xml" ContentType="application/vnd.openxmlformats-officedocument.spreadsheetml.sharedStrings+xml"/>`

File: fastexcel-reader/src/test/java/org/dhatim/fastexcel/reader/SimpleReaderTest.java (hunk 2 above)
Comment (mbrocchieri) [reply fastexcel-reader/src/test/java/org/dhatim/fastexcel/reader/SimpleReaderTest.java:140]: I can see it...

**Pull request merged**